src/util.jl

```diff
@@ -47,6 +47,19 @@ function default_example()
     joinpath(examples_dir(), "bbm_bbm_1d", "bbm_bbm_1d_basic.jl")
 end
 
+"""
+    data_dir()
+
+Return the directory where the data files provided with DispersiveShallowWater.jl are located. If DispersiveShallowWater is
```

Comment:
```suggestion
Return the directory where the data files provided with DispersiveShallowWater.jl are located. If DispersiveShallowWater.jl is
```

(since you used `.jl` when mentioning the package for the first time in this docstring).